src/aqua/installation-spell/src/aqua/deploy.aqua

```diff
@@ -48,6 +47,7 @@ func deploy_single_worker(
         catch e:
             -- Create new spell, 'worker_def_cid' will be passed in init_args
             spell_id <- PeerSpell.install(air, init_args, trigger_config)
+            Srv.add_alias("worker-spell", spell_id!)
```

Comment:
Why are you doing `!` without checking? It will crash without any sane error message

Reply:
PeerSpell.install returns string, not ?string

Reply:
Then why do you need `!`?

Reply:
Move it to separate variable, use, and only after everything succeeds – write to stream